corehq/apps/builds/static/builds/js/edit-builds.js

```diff
@@ -94,6 +94,17 @@ hqDefine('builds/js/edit-builds', [
     var buildsMenu = menuModel();
     $("#menu-form").koApplyBindings(buildsMenu);
 
+    function legacySupportViewModel() {
+        var self = {};
+        self.legacy_support_checked = ko.observable(false);
+
+        self.toggle_show_legacy = function() {
+            self.legacy_support_checked(!self.legacy_support_checked());
+        };
+        return self;
+    }
+    $("#legacy-support").koApplyBindings(new legacySupportViewModel());
```

Comment:
You don't need the `new` since this "class" is just a function.

As a naming convention, functional classes (and other classes, I suppose, although most of our classes are functional) get capitalized: `LegacySupportViewModel`